experimental/rocm/rocm_event.c

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "experimental/rocm/rocm_event.h"
```

Comment:
I wish we'd given more thought to this directory tree, as these include paths are unfortunate. I feel there should be an extra `iree` in there somewhere (experimental/rocm/iree/rocm_event.h`). If this is going to be long lived in the experimental tree, it is something to fix so the path remains globally unique. If this is on a short graduation, we can let it be this way for a bit. In either case, a followup is fine.